### ISSUE TITLE
Not getting the conference logs when conference is created by dial()

### ISSUE THREAD

**Issue by stephanrotolante** [twilio/twilio-node#604]
### Issue Summary
A summary of the issue and the environment in which it occurs. If suitable, include the steps required to reproduce the bug. Please feel free to include screenshots, screencasts, or code examples.

I am not able to see the conference I just created using the dial() function. I need this so I can grab the conference sid off of it to make another outbound call with twilio client. But the ironic thing is that the conference I create using dial() appears in the conference logs on twilio's website. Another weird thing... I am only time I am able to see conference logs using twilio client is when I create the conference through twilio client.

### Steps to Reproduce
1. This is the first step
2. This is the second step
3. Further steps, etc.

### Code Snippet
```
const conferenceID: string = uuid();

      const VoiceResponse = new twiml.VoiceResponse();

      VoiceResponse.dial().conference(
        {
          maxParticipants: 2,
          startConferenceOnEnter: true,
          endConferenceOnExit: true,
        },
        conferenceID
      );

      setTimeout(async () => {
        let data = await client.conferences.list({
          limit: 1,
          friendlyName: conferenceID,
          status: 'in-progress',
        });

       // I should be seeing the newly created conference here
        console.log(data);
      }, 5000);

      return h
        .response(VoiceResponse.toString())
        .code(200)
        .type('application/xml');
```

### Technical details:
* twilio-node version: 
* node version:

**Comment by jdegger** [comment]
I'm having issues with conference participants right now. Are you also getting a JSON parse error?

**Comment by stephanrotolante** [comment]
@jdegger Yes, I got a JSON parse issue last night when I tried to add a participant to a conference.

**Comment by jdegger** [comment]
Twilio is having an outage: https://status.twilio.com/

**Comment by childish-sambino** [comment]
Was this resolved as part of the outage, or is it still an issue?

I've been unable to recreate the issue locally using basically your exact code. Perhaps the caller is hanging up before 5 seconds?

**Comment by jdegger** [comment]
@childish-sambino the issue has been resolved for me, not sure about @stephanrotolante of course.

However I would like to suggest to implement more graceful errors when the API is returning 5xx errors. Those errors seem to be formatted in text/html and not in application/json which causes the JSON.parse function to bug out. It would have been quite easy to display the 500 error in the errors/stacktrace which would have made finding this issue much easier.

Let me know if something like that is feasible.

**Comment by stephanrotolante** [comment]
@childish-sambino Yes, looks like it is resolved. Thank you for taking time to re-create!
